### PR TITLE
dev: update docker

### PR DESCRIPTION
**Pull request by greged93**
Time spent on this PR: 0.1 day

Resolves: #NA

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?
Push the hive genesis binary in the rpc image.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docker/rpc/Dockerfile [hunk 1]
```diff
@@ -61,10 +61,11 @@ RUN build_platform() { \
     fi; \
     echo "Using BINDGEN_EXTRA_CLANG_ARGS: $BINDGEN_EXTRA_CLANG_ARGS"; \
     # Build the Rust application for the specified target
-    BINDGEN_EXTRA_CLANG_ARGS=$BINDGEN_EXTRA_CLANG_ARGS cargo build --all --release \
+    BINDGEN_EXTRA_CLANG_ARGS=$BINDGEN_EXTRA_CLANG_ARGS cargo build --all --all-features --release \
```
Comment (Eikix) [inline docker/rpc/Dockerfile:64]: will all features build in "testing" mode?

Comment (greged93) [reply docker/rpc/Dockerfile:64]: unfortunatly yes :/ could maybe build a second time in debug mode with testing feature?

Comment (Eikix) [reply docker/rpc/Dockerfile:64]: isn't this bad for us returning the eth hash?

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**